DOC: Add note on installing without dependencies (#222)
adriangb/scikeras: docs/source/install.rst

We recommend to use a virtual environment for this.

To install without any dependencies, eg. to use a custom
TensorFlow build or `tensorflow-cpu`_, you can instruct pip to ignore dependencies:

.. code:: bash

    pip install --no-deps scikeras

> Then you can manually install any compatible TensorFlow and Scikit-Learn package.

Reviewer: ```suggestion
Then manually install compatible versions of TensorFlow and Scikit-Learn.
```

> Currently, our minium versions are:

Reviewer: This will go out of date unless a test is implemented or they're generated programmatically.

Maybe it's better to say "As of SciKeras v0.2.1, the minimum required versions are as follows:"

Author: Yeah that sounds good. I thought about something programmatic, but it gets complicated obviously. I think your text will do for now.
